client/templates/frontlayout.html

```diff
@@ -0,0 +1,7 @@
+<template name="frontLayout">
+  <div>
+    {{> header}}
+
+    {{> frontBody}}
```

Comment:
> This should have been a template dynamic so that the "main" parameter can be passed in by flow layout!
You should do sth lke:
```
{{> Template.dynamic template=main}}
```
instead of {{> frontBody}}